assets/images/Svg.js

```diff
@@ -59,7 +59,8 @@ const svg = {
   nextjs: require("./nextjs.svg"),
   showSources: require("./showSources.svg"),
   showOutline: require("./showOutline.svg"),
-  nuxtjs: require("./nuxtjs.svg")
+  nuxtjs: require("./nuxtjs.svg"),
+  rxjs: require("./rxjs.svg")
```

Comment:
interesting, why are we adding this back in?

Reply:
We have an issue which was looking to render the `rxjs` svg and couldn't find it so it return `undefined` which broke rendering. The svg file but it was not included in the map above.

Reply:
Not sure why the tests are failing ..

Reply:
no worries. i've got it